pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -1427,7 +1428,7 @@ public void closeFailed(ManagedLedgerException exception, Object ctx) {
     @VisibleForTesting
     public CompletableFuture<List<String>> getReplicationClusters(TopicName topicName) {
         CompletableFuture<Optional<TopicPolicies>> future = new CompletableFuture<>();
-        if (isSystemTopic()) {
+        if (isSystemTopic() || HEARTBEAT_TENANT.equals(topicName.getTenant())) {
```

Comment:
Can we include heartbeat topic in SystemTopic?

Reply:
done @Jason918